tasks/e2e-installs.sh

```diff
@@ -101,11 +101,11 @@ npx create-react-app --version
 # ******************************************************************************
 
 cd "$temp_app_path"
-npx create-react-app test-app-dist-tag --scripts-version=@latest
```

Comment:
Revert all changes in this file and keep it consistent with cra5
Tao Fang is working on how to build and test with CRA5